category/category.go

```diff
@@ -162,7 +163,7 @@ func (m *GormRepository) LoadAllRelationshipsOfCategory(ctx context.Context, cat
 		log.Error(ctx, map[string]interface{}{
 			"category_id": categoryID,
 		}, "category not found")
-		return nil, errors.NewBadParameterError("categoryID", categoryID)
+		return nil, errs.Wrap(err, fmt.Sprintf("Fail to load category with id %s", categoryID))
```

Comment:
In most cases we use lower-case error messaged and past tense. so `"failed to load category with id %s"` would be a better message.